### ISSUE TITLE
[Bug]: special character in user name results in engine crash with plots

### ISSUE THREAD

**Issue by jmuehlhans** [jasp-stats/jasp-issues#1758]
### JASP Version

0.16.3

### JASP Module

jaspDescriptives, jaspRegression, jaspTTests

### What analysis are you seeing the problem on?

Any plot in any module

### What OS are you seeing the problem on?

Windows 11

### Bug Description

Hi everybody,

all of my calculations in JASP work fine as long as I don't use any plots. As soon as I chosse to display descriptive plots or raincloud plot in T-Tests,  distribution plot in descriptives and so on, it will display a white window with heading and as soon as I try to scale it, the error message is dropped "The engine crashed while trying to run this analysis". 
Even more interesting: When I add a second type of plot, the first one will eventually display, sometimes for all of the items, sometimes for 1/4, however, again as soon as I try doing anything with it, the error message drops again. Tried my own data file and also the famous "kitchen rolls" testfile, same error.
I've read similar problems here already, but the solutions did not work for me (Region settings for instance).

I'd be glad about any help!

PS: First occurred to me in 0.16.2, now I was hoping to solve it with 0.16.3, which is not the case.

### Expected Behaviour

In versions prior to 0.16.2 plots would always display correctly and don't drop any error message when e.g. trying to scale them

### Steps to Reproduce

1. Load any variable in any test,
2. Check any analysis that is a plot,
3. Scale the window or try "edit image" and hit "ok"
...

### Log (if any)

[JASP 2022-06-22 12_27_15 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/8956759/JASP.2022-06-22.12_27_15.Desktop.log)
[JASP 2022-06-22 12_27_15 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/8956760/JASP.2022-06-22.12_27_15.Engine.0.log)
![Screenshot 2022-06-22 122627](https://user-images.githubusercontent.com/107992491/175007987-fa2e3d3b-b617-4dda-93e0-0ef8e2022ce6.png)

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by jmuehlhans** [comment]
Hi @boutinb I have checked that folder and it remains empty when I attempt to plot something. When I click different types of plots, eventually a graph will show up and I can see the png pop up in that folder but also disappear within a fraction of a second. Can't even catch a screenshot that fast. 
I was thinking about problems with the rights to write to that folder? Tried starting JASP as admin (even though I'm admin on that account anyway) but that won't work either.
Thanks for your help!

**Comment by shun2wang** [comment]
@jmuehlhans hi , I noticed from the log file that there is a special character in your user name. can you try this again under a user name consisting of ascii characters ?

**Comment by jmuehlhans** [comment]
Hi @shun2wang and @boutinb, 
I have created an additional admin account using no special characters and tried to replicate the error but **it worked fine**. Plots were immediately displayed, I could scale or alter them and \AppData\Local\JASP\temp\11156\resources\0 showed all of the pngs and other files. Thank you so much for that solution.

Summary:
I used a microsoft account to log into my Windows 11 computer, which had "ö" and "ü" in the name, changed that to "oe" and "ue" but my windows would after several restarts not apply the changes. So I created a new user account without special characters and JASP would not drop that error message again in any module and analysis.
